README.md

```diff
@@ -63,6 +63,35 @@ The following sections/settings are included.
 }
 ```
 
+### EPP Templates
```

Comment:
I would prefer to see a native puppet function that takes all these parameters.  Something like

```
extlib::puppet_managed(String $message = 'WARNING This file is managed by puppet. Do not edit!',
                                         String  $begin_line = '#',
                                         String  $end_line = '',
                                         Int     $line_length = 70,
                                         String  $metadata_title = 'Metadata:',
                                         Hash    $metadata = {} ) >> String {

}

# process content here and break into multiple lines
```

Then in the template it would be 

`<%= extlib::puppet_managed() %>`